### ISSUE TITLE
Neovim error with oracle

### ISSUE THREAD

**Issue by al3xfischer** [tpope/vim-dadbod#88]
I'm running NVIM v0.5.0-dev+1157-g0ab88c2ea on Win10 in Powershell and setup up vim-dadbod to access an oracle DB. (I also tried the lasted nightly build)

But I get this strange error which is saying the "<" operator is reserved for future versions.
The exact same setup works in vim just fine.

The original error message is German but means basically what I described above.

I just executed the following command.
```vim
:DB oracle://<user>:<pwd>@localhost/orcl
```

```powershell
In Zeile:1 Zeichen:49

+ sqlplus -S -L "<user>/<pwd>@localhost/orcl" < "C:\Users\XYZ\AppD ...

+ ~

Der Operator "<" ist für zukünftige Versionen reserviert.

+ CategoryInfo : ParserError: (:) [], ParentContainsErrorRecordException

+ FullyQualifiedErrorId : RedirectionNotSupported
```

**Comment by Rom1deTroyes** [comment]
Just to be sure, did you replace `<user>` and `<pwd>` by your credentials ?
I mean, the final command don't have <kbd><</kbd> and <kbd>></kbd> characters, something like :
```
:DB oracle://myusername:mypassword@localhost/orcl
```
(sometimes I quickly forget obvious things like that :-))

**Comment by al3xfischer** [comment]
@Rom1deTroyes  Yes, my command didn't include `<` and `>` characters, I just replaced it in the markdown.  
My actual call looked similar to the one you mentioned.

**Comment by tpope** [comment]
PowerShell is such a pain in the ass. I think this is a wontfix until after #64 gives us the option to avoid the shell entirely. @kristijanhusak I guess this means we'll want to pursue using `ch_sendraw()`/`chansend()` to write to stdin directly.

**Comment by kristijanhusak** [comment]
Would using `cmd` instead of PowerShell make it work? Just curious, I don't have much experience with Vim on windows.

**Comment by tpope** [comment]
It absolutely would. It's becoming an increasingly popular default, though. "PowerShell isn't supported" is starting to sound as out of touch as "zsh isn't supported". I wouldn't say it's worth bending over backwards for, but given that we want to get rid of the shell anyways...

**Comment by kristijanhusak** [comment]
I thought so, just wanted to check. It would be strange that this kind of bug comes in so late. I'll try to figure something out during the weekend.

**Comment by al3xfischer** [comment]
@kristijanhusak  If I run it in the classic `cmd` I get the same error message.

**Comment by tpope** [comment]
@al3xfischer you can try the async-query branch in #64 and see if that works for you.

**Comment by al3xfischer** [comment]
@tpope  
I tried it.

If I run the following no error messages are displayed and a new buffer gets opend.
```shell
:DB oracle://myusername:mypassword@localhost/orcl
```

If I run a simple query I get the same error message.
```shell
:DB oracle://myusername:mypassword@localhost/orcl select 1 from dual
```

**Comment by kristijanhusak** [comment]
@al3xfischer completely same? That would be really strange, since ` < ` is no longer used anywhere in the code on #64 .

**Comment by al3xfischer** [comment]
@kristijanhusak  see below. I used the above-mentioned command.

```
 In Zeile:1 Zeichen:49
  + sqlplus -L -S "myusername/mypassword@localhost/orcl" < "C:\Users\XYZ\AppD ...
  +  ~
  Der Operator "<" ist für zukünftige Versionen reserviert.
      + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
      + FullyQualifiedErrorId : RedirectionNotSupported
```

May I got something wrong, but I used [this](https://github.com/kristijanhusak/vim-dadbod/tree/async-query) repo.
I also made sure that my local version does not use the original vim-dadbod repo.
Did I miss anything?

**Comment by kristijanhusak** [comment]
@al3xfischer I think you didn't install it properly. What plugin manager are you using? Give me the chunk of vimrc where you set up vim-dadbod.

**Comment by al3xfischer** [comment]
I use vim-plug.

Plug 'kristijanhusak/vim-dadbod'

Ahhh... is it that I use the wrong branch?

**Comment by kristijanhusak** [comment]
Yes, it should be like this:
```
Plug 'kristijanhusak/vim-dadbod', { 'branch': 'async-query' }
```

**Comment by al3xfischer** [comment]
And of course it works! 

Sry my bad.
Thanks for the support!
